docs/en/configuration/source-plugins/neo4j.md

```diff
@@ -0,0 +1,133 @@
+# Source plugin: Neo4j
+
```

Comment:
Please change the title `Source plugin: Neo4j` to `Source plugin: Neo4j [Spark]`

Reply:
it was fixed